### ISSUE TITLE
[Docs] Error in RBFKernel documentation

### ISSUE THREAD

**Issue by mc-robinson** [cornellius-gp/gpytorch#752]
I don't believe the documentation is technically correct for `gpytorch.kernels.RBFKernel`. The documentation says that the covariance matrix is computed using:

\begin{equation*}
          k_{\text{RBF}}(\mathbf{x_1}, \mathbf{x_2}) = \exp \left( -\frac{1}{2}
          (\mathbf{x_1} - \mathbf{x_2})^\top \Theta^{-1} (\mathbf{x_1} - \mathbf{x_2}) \right)
       \end{equation*}

    where :math:`\Theta` is a :attr:`lengthscale` parameter.

However, consider the 1D case where x1 and x2 are scalars, and `gpytorch/functions/rbf_covariance.py` is called in the calculation,  the following code is then called:

```
x1_ = x1.div(lengthscale)
x2_ = x2.div(lengthscale)
unitless_sq_dist = sq_dist_func(x1_, x2_)
```

Thus, the denominator is actually `\Theta^2` and not the lengthscale `\Theta` as currently written in the documentation. Since I believe that the lengthscale tensor must be diagonal (not totally positive about this), I think the denominator can just be written as `\Theta^{-2}` to maintain generality. 

Happy to make a pull request if that looks alright to you.

**Comment by jacobrgardner** [comment]
Yeah, that is indeed a mistake! it's actually divided by `2 * ls^2`, with the 2 coming from the postprocess function:
https://github.com/cornellius-gp/gpytorch/blob/338d93fdc829ff27f03bbfa3c83f7066f16b95a7/gpytorch/kernels/rbf_kernel.py#L8

A pull request would definitely be welcome! If you don't get around to putting one up I'll be able to tomorrow.

**Comment by mc-robinson** [comment]
Great, just submitted pull request. As far as I can tell, the factor of (-1/2) is already included in the documentation. But I may be missing something

**Comment by jacobrgardner** [comment]
closed by #753
